commons/active/banking/banking_attempt.avsc

```diff
@@ -0,0 +1,30 @@
+{
+  "namespace": "org.radarcns.active.banking",
+  "type": "record",
+  "name": "BankingAttempt",
+  "doc": "A Banking App Attempt to complete a transaction, which can contain a PIN, an Amount and a Confirm/Cancel Transaction event. It can also time out or end.",
+  "fields": [
+    { "name": "uid", "type": "string", "doc": "Unique ID for an Attempt."},
+    { "name": "time", "type": "double", "doc": "Time since the Unix Epoch (seconds)." },
+    { "name": "timeCompleted", "type": "double", "doc": "Time since the Unix Epoch (seconds)." },
+    { "name": "duration", "type": "double", "doc": "Duration of the activity (seconds)."},
```

Comment:
Is this a proxy for `duration = timeCompleted - time`? Perhaps this variable can be removed then.